spec.md

```diff
@@ -2250,7 +2386,7 @@ f("n")                                          # 2
 ### Index expressions
 
 An index expression `a[i]` yields the `i`th element of an _indexable_
```

Comment:
"Element" needs some clarification here. A bytes element is a byte, but there is no datatype called a "byte". We can also note that `a[i]` is equal to `a[i:i+1]` for strings alone.

Reply:
The bytes type is defined as:

> A _bytes_ is an immutable sequence of values in the range 0-255.

and shortly after, it says:

> The index expression `b[i]` returns the int value of the ith element.